### PR TITLE
Fix: lint warning fix for any to unknown #1374

### LINKED ISSUE #1374: style: 2021-09-20 linter warnings batch 25 / 26
style: 2021-09-20 linter warnings batch 25 / 26

### Description

Fix the linter errors in the Fabric Test Ledger class (`packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts`)

### Acceptance Criteria
1. Each linter warning is either fixed or a specific new issue is created aimed at exactly that one linter warning where applicable: If fixing a linter warning requires a larger refactoring of the codebase, then it is justifiable to break it out into it's own GH task.
2. The pull request submitted links to all (if any) additional issues that were created for the linter warnings that did not fit in the scope of the issue.

### Full List of Linter Warnings

packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts
   190:52  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   292:52  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   396:67  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   535:14  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   588:28  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   644:30  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   776:25  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   830:29  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   866:34  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  1319:15  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

### PR DESCRIPTION
**Pull request by yayoiukai2021**
Lint fix for batch 25/26. + No.251 and No.240 (In the same file) 

Fixes #1374

### CODE REVIEW

File: packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts [hunk 1]
```diff
@@ -101,14 +101,14 @@ export class QuorumTestLedger implements ITestLedger {
   }
 
   public async getFileContents(filePath: string): Promise<string> {
-    const response: any = await this.getContainer().getArchive({
+    const response: unknown = await this.getContainer().getArchive({
```
Comment (petermetz) [inline packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts:104]: @yayoiukai2021 

I recommend simply omitting the explicit type declaration here in order to make the linter warning go away. Using unkown in this particular case just makes the compiler fail somewhere down the line.

```suggestion
const response = await this.getContainer().getArchive({
```

**Review by petermetz — CHANGES_REQUESTED** [review]

**Comment by petermetz** [comment]
@yayoiukai2021 The other thing apart from my recommended change above is the DCO: If you click on the link labeled as "Details" next to the failing check (DCO) then it should give you specific instructions on how to fix it on the command line. If you encounter any trouble with it then feel free to ask for more help ;-)

**Review by petermetz — APPROVED** [review]
@yayoiukai2021 LGTM, thank you very much!

**Review by takeutak — APPROVED** [review]

File: packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts [hunk 2]
```diff
@@ -372,25 +372,28 @@ export class QuorumTestLedger implements ITestLedger {
...
+                if (progressError) {
+                  reject(progressError);
+                } else {
+                  resolve(output);
+                }
+              },
+            );
+          }
+        },
+      );
```
Comment (petermetz) [inline packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts:396]: @yayoiukai2021 I'd fix the compiler error like this:

```suggestion
  private pullContainerImage(containerNameAndTag: string): Promise<unknown[]> {
    return new Promise((resolve, reject) => {
      const docker = new Docker();
      docker.pull(containerNameAndTag, (pullError: unknown, stream: Stream) => {
        if (pullError) {
          reject(pullError);
        } else {
          docker.modem.followProgress(
            stream,
            (progressError: unknown, output: unknown[]) => {
              if (progressError) {
                reject(progressError);
              } else {
                resolve(output);
              }
            },
          );
        }
      });
    });
  }
```

**Review by petermetz — CHANGES_REQUESTED** [review]
@yayoiukai2021 You will also need to import the `Stream` type in order to make my other suggestion work with something like this:

```typescript
import { Stream } from "stream";
```

**Comment by petermetz** [comment]
Closing due to inactivity, feel free to reopen as needed and then we can continue the review.

**Pull request closed without merging**